src/matcha_ml/runners/base_runner.py

```diff
@@ -161,6 +162,11 @@ def _destroy_terraform(self, msg: str = "") -> None:
             if ret_code != 0:
                 raise MatchaTerraformError(tf_error=err)
 
+    def _clean_up(self) -> None:
```

Comment:
Is `_clean_up` function used by both `AzureRunner` and `RemoteStateRunner`?

Reply:
Don't think, will move this to `RemoteStateRunner` instead. Good spot.